src/Npgsql/Properties/NpgsqlStrings.resx

```diff
@@ -75,4 +75,7 @@
     <data name="FullTextSearchNotEnabled" xml:space="preserve">
         <value>Full-text search isn't enabled; please call {0} on {1} to enable full-text search.</value>
     </data>
+    <data name="RangesNotEnabled" xml:space="preserve">
```

Comment:
The check can now be added to the UnsupportedTypeInfoResolver

Reply:
Done